xml/update_caasp.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="update-rook">
+ <!-- ============================================================== -->
+ <title>Updating &rook;</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This chapter describes how to update containerized &productname;
+  &prevproductnumber; on top of a &caasp; 4.5 &kube; cluster.
+ </para>
+ <para>
+  This chapter takes you through the steps to update the software in a
+  &rookceph; cluster from one version to the next. This includes both the
+  &rookceph; operator software itself as well as the &ceph; cluster software.
+ </para>
+ <note>
+  <para>
+   Version
+  </para>
+ </note>
+ <sect1 xml:id="rook-update-hardware-specs">
+  <title>Recommended hardware specifications</title>
+
+  <para>
+   For &productname; deployed with &rook;, the minimal configuration is
+   preliminary, we will update it based on real customer needs.
+  </para>
+
+  <para>
+   For the purpose of this document, consider the following minimum
+   configuration:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     A highly available &kube; cluster with 3 master nodes
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Four physical &kube; worker nodes, each with two OSD disks and 5 GB of RAM
+     per OSD disk
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Allow additional 4 GB of RAM per additional daemon deployed on a node
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Dual-10 Gb ethernet as bonded network
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     If you are running a hyper-converged infrastructure (HCI), ensure you add
+     any additional requirements for your workloads.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="patch-release-upgrades">
+  <title>Patch release upgrades</title>
+
+  <para>
+   To update a patch release of &rook; to another, you need to update the
+   common resources and the image of the &rook; operator.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Get the latest common resource manifests that contain the relevant changes
+     to the latest version:
+    </para>
+<screen>
+&prompt.user; git clone --single-branch --branch <replaceable>version-number</replaceable> https://github.com/rook/rook.git
```

Comment:
This would be applying the upstream containers I believe. We should get the latest rook-k8s-yaml package as per the installation. However I believe the install only really covers helm.

Reply:
Correct! Customers need to use our rpm-packages for the upgrades. Either as in this case rook-k8s-yaml or the helm-chart package if customers are using helm.